src/lesson34/js/article.js

```diff
@@ -140,6 +142,28 @@ const createThumbnail = data => {
 
 const renderCategory = data => document.querySelector('.js-article-info').insertAdjacentElement('beforeend', createCategoryLabel(data));
 
+const changeButtonDisabled = target => {
+    const starImage = target.firstElementChild;
+    target.disabled = true;
+    starImage.src = '/assets/img/icon-star-done.png';
+}
+
+const saveArticleData = data => {
+    const targetData = getArticleData(data);
+    const favoriteArticleData = {
+        [targetData.id]: {
+            'date': targetData.date,
```

Comment:
ここにidを持った方が今後参照する時に楽になると思いますよ
```suggestion
            'id', targetData.id,
            'date': targetData.date,
```